wikimedia/wikipedia-ios
# Fix scrollview inset for On This Day VC

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T264932

### Notes
* Further adventures in "we actually have almost everything we need in our custom NavBar, I just need to learn how to use all the features". This flag fixes the bad scrolling problem.

### Test Steps
1. Add On This Day widget to your test device/sim.
2. Tap the widget to open the app into the On This Day VC.
3. Ensure the historical item tapped is appropriately placed on the screen - and with the largest circle on the timeline.

### Screenshots/Videos
https://user-images.githubusercontent.com/9295855/102942184-8c3d1400-4469-11eb-928f-f921aa293133.mov
